views/js/front/orderDetail.js

```diff
@@ -1,16 +1,18 @@
+function toggleModal() {
```

Comment:
doesn't exists in bootstrap ?

Reply:
don't think you can use bootstrap on prestashop front shop. On the Admin side yes but not on the front

Reply:
so does the flex as been tested on Internet Explorer ?

Reply:
No it was not, but after reading a few articles, I can say that there shouldnt be any problem with the flex property. IE has some issues with more complicated properties of flexbox that we are not using.

Reply:
test on IE!